rotkehlchen/data_import/importers/cryptocom.py

```diff
@@ -441,6 +479,8 @@ def _import_cryptocom_associated_entries(
                     )
                     self.add_trade(write_cursor, trade)
 
+                self.imported_entries += 1 + len(debited_rows)
```

Comment:
add a small comment why this is so. I can't understand why by quickly looking at it.